segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageMockTests.java

```diff
@@ -339,73 +338,6 @@ public void testIOExceptionDuringWrite() throws Exception {
         //verify(spyChunkStorage, times(1)).doDelete(any());
     }
 
-    @Test
-    public void testFileNotFoundExceptionDuringGarbageCollection() throws Exception {
```

Comment:
Was this deleted on purpose or is it temporary too?

Reply:
redundant test, covered elsewhere